### PR TITLE
Prevent double submission of form

### PR DESCRIPTION
**Pull request by stevenwilkin**
Alternative take on #165

### CODE REVIEW

File: app/assets/javascripts/preview.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+$(function () {
```
Comment (andrewgarner) [inline app/assets/javascripts/preview.js]:
```js
(function ($) {
  'use strict';

})(jQuery);
```

File: app/views/appointment_summaries/preview.html.erb [hunk 2]
```diff
@@ -16,3 +16,7 @@
     </div>
   </div>
 </div>
+
+<% content_for :body_end do %>
+  <%= javascript_include_tag 'preview' %>
```
Comment (gjvis) [inline app/views/appointment_summaries/preview.html.erb]: Better to include in the main application.js payload I think, its a very small piece of script (and is likely to remain so).

Comment (stevenwilkin) [inline app/views/appointment_summaries/preview.html.erb]: Fair enough

File: app/assets/javascripts/preview.js (hunk 1 above)
Comment (stevenwilkin) [inline app/assets/javascripts/preview.js]: Good spot

Comment (stevenwilkin) [inline app/assets/javascripts/preview.js]: Not congruent with `init.js` though!

**Comment by andrewgarner** [comment]
Did you review the 'Rails way' that is using `disable_with`? - http://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html#method-i-button_tag-label-Data+attributes

**Comment by stevenwilkin** [comment]
Every day's a school day @andrewgarner ! That does the job very neatly

**Comment by stevenwilkin** [comment]
We good to go with this `disable_with` iteration then?

**Comment by gjvis** [comment]
yes

**Comment by gjvis** [comment]
+2 −1 wins

**Comment by andrewgarner** [comment]
You need `rails/jquery-ujs` for this to work!

**Comment by stevenwilkin** [comment]
[govuk_admin_template pulls in ujs](https://github.com/alphagov/govuk_admin_template/blob/7d480bb54abe162fa45b0275939fa8f8e625a375/app/assets/javascripts/govuk-admin-template.js#L2) @andrewgarner

**Comment by andrewgarner** [comment]
We're all fine and dandy then :smile:

**Pull request merged**